src/actions/databases.js

```diff
@@ -1,4 +1,4 @@
-const getDB = require('remote').require('./src/db').getDB;
+import { getDBSession } from '../db';
```

Comment:
about the name, "session" looks a bit strange for me... perhaps we can use getDBConnection or some other thing, it's more a database connection or connection instance than a session...

what do you think?